src/client/sandbox/code-instrumentation/methods.js

```diff
@@ -1,12 +1,46 @@
 import SandboxBase from '../base';
 import { isNullOrUndefined, inaccessibleTypeToStr, isWindow, isDocument } from '../../utils/types';
 import { DOCUMENT_WRITE_BEGIN_PARAM, DOCUMENT_WRITE_END_PARAM, CALL_METHOD_METH_NAME } from '../../../processing/js';
+import { FOCUS_PSEUDO_CLASS_ATTR } from '../../../const';
+import { isDomElement } from '../../utils/dom';
```

Comment:
Maybe it would be better to move `is*`-family functions (`isDomElement`, `isAnchor`, `isInput` and so on)  from `utils/dom` to `utils/types`? So we will have hub module for element type checking, like  we have the module `Browser` for browser checking.